.github/workflows/ci.yml

```
@@ -1,8 +1,8 @@
 name: '@ngworker/lumberjack CI'
 
 env:
-  ANGULAR_LATEST: 11.1.x
-  NODE_LATEST: 12.x
+  ANGULAR_LATEST_STABLE: 11.2.x
+  NODE_LATEST_STABLE: 12.x
```

This name is no better than `NODE_LATEST` since we know it's neither the latest, nor the latest stable. Either roll back this change or find a name that expresses that it's the latest Node.js version supported by the latest stable version of Angular.

---

Also add this comment to make it more clear
```suggestion
  # The latest version supported by the Angular version listed in `ANGULAR_LATEST_STABLE`.
  NODE_LATEST_STABLE: 12.x
```

---

NODE_DEFAULT ??

---

We could also delete these variables if they are too confusing but we would lose some speed because we can't restore the exact cache.